provisioning/miniprov/hare_mp/cdf.py

```diff
@@ -275,11 +275,13 @@ def _create_node(self, machine_id: str) -> NodeDesc:
         # Adding a Motr confd entry per server node in CDF.
         # The `runs_confd` value (true/false) determines if Motr confd process
         # will be started on the node or not.
-        servers.value.append(M0ServerDesc(
-            io_disks=DisksDesc(
-                data=DList([], 'List Text'),
-                meta_data=Maybe(None, 'Text')),
-            runs_confd=Maybe(True, 'Bool')))
+        start_confd = store.get(f'server_node>{machine_id}>consul_server')
+        if start_confd == 'True':
```

Comment:
```suggestion
        if start_confd.lower() == 'true':
```
End users are very inventive.

Reply:
BTW I would suggest to improve `conf.get()` method so that it can optionally perform the type transformation. That would help to localize this kind of "boolean" checks. Especially if the rules how to transform a string to a proper bool value change (e.g. I can imagine that "1" is also meant as `True`, why not? Type rules are not defined in ConfStore specification, so there are so many things that can be understood differently...

Reply:
Agree, actually I think we can get consul server information without conf-store through `consul members` output, checking that.

Reply:
Not sure it is a good way to go. At least that would require consul cluster to exist at the moment when CDF is being generated. And from the practical point of view - it is hard to run mini provisioner scripts already: we need to have conf-store JSON file, we need `/etc/machine` file to exist and match to our conf-store file. If we add Consul into the picture, it will become even more complicated. And how someone would say - too coupled 😉

Reply:
It is expected that Consul is running already as a3rd party component, We will anyway be not starting Consul but Hare has to use it in init anyway.

Reply:
Still, if it is possible to keep this information in ConfStore plane only, I would vote for not involving Consul into the picture.

Reply:
Okay, let me see how bool values are used in conf-store.

Reply:
Used existing conf-store key `server_node>{machine-id}>roles` instead of bool. The same key is used for kafka and openldap servers.